src/plugins/condition/components/Condition.vue

```diff
@@ -291,12 +292,12 @@ export default {
         },
         removeCriterion(index) {
             this.condition.configuration.criteria.splice(index, 1);
-            this.persist()
+            this.persist();
         },
         cloneCriterion(index) {
-            const clonedCriterion = {...this.condition.configuration.criteria[index]};
+            const clonedCriterion = _.cloneDeep(this.condition.configuration.criteria[index]);
```

Comment:
`const clonedCriterion = Object.assign({}, this.condition.configuration.criteria[index]); `
could also work here if you don't want to invoke `lodash`.

Reply:
Again, Object.assign failed because it didn't do a deep copy. The ES6 spread operator does the same as Object.assign, failing to clone deeply enough.

Reply:
correct. object.assign will not work for deep copies